linters/ruff/ruff_to_sarif.py

```diff
@@ -6,15 +6,15 @@
 results = []
 
 
-def get_region(entry):
```

Comment:
The one piece I was missing--is this issue you were describing just specific to ruff/to this parser? If so, can you add a comment before line 59 and maybe before line 9 briefly describing the reason for this off-by-one fix?

Reply:
This is something specific to Ruff. Just added comments